lib/iex/lib/iex/helpers.ex

```diff
@@ -325,9 +325,26 @@ defmodule IEx.Helpers do
 
   @doc """
   Retrieves the nth expression's value from the history.
+  Where nth is iex(nth)>
 
   Use negative values to look up expression values relative to the current one.
+
```

Comment:
I think we should not have this newline, `v(-1)` example belongs to the previous sentence.